subgraphs/_reference_/src/prices/config/arbitrum.ts

```diff
@@ -0,0 +1,131 @@
+///////////////////// CALCULATIONS/ORACLE CONTRACT ////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const YEARN_LENS_CONTRACT_ADDRESS = Address.fromString(
+  "0x043518ab266485dc085a1db095b8d9c2fc78e9b9"
+);
+export const CHAIN_LINK_CONTRACT_ADDRESS = Address.fromString(
+  "0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf"
+);
+export const AAVE_ORACLE_CONTRACT_ADDRESS = Address.fromString(
+  "0xb56c2F0B653B2e0b10C9b928C8580Ac5Df02C7C7"
+);
+export const SUSHISWAP_CALCULATIONS_ADDRESS = Address.fromString(
+  "0x5ea7e501c9a23f4a76dc7d33a11d995b13a1dd25"
+);
+
+///////////////////////////////////////////////////////////////////////////
+///////////////////////////// CURVE CONTRACT //////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const CURVE_CALCULATIONS_ADDRESS = Address.fromString(
+  "0x3268c3bda100ef0ff3c2d044f23eab62c80d78d2"
+);
+
+export const CURVE_REGISTRY_ADDRESSES: Address[] = [
+  Address.fromString("0x445FE580eF8d70FF569aB36e80c647af338db351"),
+  Address.fromString("0x0E9fBb167DF83EdE3240D6a5fa5d40c6C6851e15"),
+];
+
+///////////////////////////////////////////////////////////////////////////
+/////////////////////////// UNISWAP FORKS CONTRACT ////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const UNISWAP_FORKS_ROUTER_ADDRESSES: Address[] = [
+  Address.fromString("0x1b02da8cb0d097eb8d57a175b88c7d8b47997506"), // SushiSwap
+];
+
+///////////////////////////////////////////////////////////////////////////
+/////////////////////////// BLACKLISTED TOKENS ////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const YEARN_LENS_BLACKLIST: Address[] = [];
+export const AAVE_ORACLE_BLACKLIST: Address[] = [];
+export const CURVE_CALCULATIONS_BLACKSLIST: Address[] = [];
+export const SUSHI_CALCULATIONS_BLACKSLIST: Address[] = [];
+
+///////////////////////////////////////////////////////////////////////////
+//////////////////////////// HARDCODED STABLES ////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const HARDCODED_STABLES: Address[] = [];
+
+///////////////////////////////////////////////////////////////////////////
+///////////////////////////////// HELPERS /////////////////////////////////
+///////////////////////////////////////////////////////////////////////////
+
+export const USDC_TOKEN_DECIMALS = BigInt.fromI32(6);
+
+export const ETH_ADDRESS = Address.fromString(
+  "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"
+);
+export const WETH_ADDRESS = Address.fromString(
+  "0x82af49447d8a07e3bd95bd0d56f35241523fbab1"
+);
+export const USDC_ADDRESS = Address.fromString(
+  "0xff970a61a04b1ca14834a43f5de4533ebddb5cc8"
+);
+
+export class config implements Configurations {
```

Comment:
not necessary to change, but one way to reduce the code and get rid of this class in every network config:
- you setup the network config as a data structure, and import that into utils.ts
- Then when the switch finds what network is being used it passes that as the constructor arg then all of the constants get set

Reply:
I have not found a way to create the data structure using a type or interface to be passed along to a constructor.